### ISSUE TITLE
Add WebAssembly framework moniker to NuGet

### ISSUE THREAD

**Issue by mhutch** [mono/mono#7839]
Add the WebAssembly framework moniker to the [NuGet client library](https://github.com/NuGet/NuGet.Client/blob/6e6f469828435d0a3c88aa153808ba380c6d0d71/src/NuGet.Core/NuGet.Frameworks/FrameworkConstants.cs#L36) so that NuGet will be able to handle packages that target this framework.

**Comment by marek-safar** [comment]
This is not right repo to ask for new TFM. I actually don't know which one it's but we used https://github.com/dotnet/standard in the past for such requests.

**Comment by mhutch** [comment]
@marek-safar this is for tracking it as part of the overall WebAssembly story

**Comment by mhutch** [comment]
PR files with NuGet at https://github.com/NuGet/NuGet.Client/pull/2159
